src/main/resources/default.properties

```diff
@@ -0,0 +1,165 @@
+# app name
```

Comment:
Is this file to replace `src/main/resources/portal.properties.EXAMPLE`? I feel that's sort of our default config now.